vars/main.yml

```diff
@@ -339,6 +341,20 @@ pgbackrest_conf:
     - {option: "process-max", value: "2"}
     - {option: "recovery-option", value: "recovery_target_action=promote"}
 #    - {option: "", value: ""}
+
+pgbackrest_srv_conf:
```

Comment:
I think a detailed comment is needed here for what the pgbackrest_srv_conf variables are for, since they are very similar to the pgbackrest_conf variables